yarn.lock

```diff
@@ -2956,14 +2985,14 @@
     react-modal "^3.15.1"
     react-transition-group "^4.4.2"
 
-"@navikt/familie-form-elements@^6.3.7":
-  version "6.3.7"
-  resolved "https://npm.pkg.github.com/download/@navikt/familie-form-elements/6.3.7/ff7317fdfca794ff0e37c5e4d665f829aad4ecac#ff7317fdfca794ff0e37c5e4d665f829aad4ecac"
-  integrity sha512-HbA73UX+UDR54lD7wTyDaB6MXUzo6X3UhJN5obZvCTeRZp5SVxS5CGsIv5pD8Bu8dJSrGtF985h4/LayuR0NSQ==
+"@navikt/familie-form-elements@^9.0.0":
+  version "9.0.0"
+  resolved "https://npm.pkg.github.com/download/@navikt/familie-form-elements/9.0.0/75ed6c1e1791849cdee97a48a39b1e93d9e62aac#75ed6c1e1791849cdee97a48a39b1e93d9e62aac"
+  integrity sha512-qGpHtVKTxCal1wisDGjVdnCQVSoM2MjGNFDsHEbSX9QekWzT4MhCLTS3e21n75e8Wd6uy08YAE0a5DU+ldDl/A==
   dependencies:
-    "@navikt/ds-css" "1.5.x"
-    "@navikt/ds-react" "1.5.x"
-    "@navikt/ds-tokens" "1.5.x"
+    "@navikt/ds-css" "2.0.x"
+    "@navikt/ds-react" "2.0.x"
+    "@navikt/ds-tokens" "2.0.x"
```

Comment:
Samme som i klage - å kjøre både css for 1.5.X og 2.0.X kan by på utfordringer

Reply:
✅ Oppdatert til ds2